code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -649,6 +649,7 @@
 
 	if(health >= 0)
 		if(src == M)
+			var/to_send = list()

```

Comment:
This isn't necessary at all, and does nothing but add unnecessary performance overhead and throw a needless list at the GC. You can use the `+=` operator with strings directly instead of going the long way around with heavy operations like jointext()

Reply:
The only reason I used it because I had to do something similar over on BayStation and it was suggested to use a list instead of concatenating it into one big string. I'll commit a change in a moment...